### DISCUSSION TITLE
Error trying to migrate Parcel 1.0 to 2.0

### DISCUSSION THREAD

**Discussion by frantwo** [parcel-bundler/parcel#8309] [Q&A]
Hello to everyone, 

It is my first time adding a new discussion here. So I apologize you if I say something wrong.
I am trying to migration one project from V1.0 to V2.0 but after follow the instructions from :
https://parceljs.org/getting-started/migration/#getting-started

I end always into the next error:
![image](https://user-images.githubusercontent.com/11053804/179220134-a87394ca-d17d-4a01-b8a6-30cef67f796d.png)

My package.json is the next:
`{
  "name": "proceed",
  "version": "2.3.6",
  "description": "",
  "scripts": {
    "start": "parcel index.html src/index.js deployCfg.js",
    "build": "parcel build index.html src/index.js deployCfg.js --no-optimize --no-cache && npm run build-cache",
    "build-cache": "node build-cache.js",
    "lint": "eslint src && prettier --check \"src/**/*.js\"",
    "lint:fix": "eslint --fix src && prettier --write \"src/**/*.js\"",
    "lint:tests": "eslint tests && prettier --check \"tests/integration/*.js\"",
    "all-tests": "cd testing/integration && npx codeceptjs run --verbose",
    "all-tests-allure": "cd testing/integration && npx codeceptjs run  --plugins allure --verbose",
    "test:new": "codeceptjs gt",
    "preinstall": "npx npm-force-resolutions"
  },
  "staticFiles": {
    "staticPath": "public",
    "watcherGlob": "**"
  },
  "browserslist": [
    "defaults"
  ],
  "repository": {
    "type": "git",
    "url": "git+ssh://git@bitbucket.org/tnt-int/proceed.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "homepage": "https://bitbucket.org/tnt-int/proceed#readme",
  "dependencies": {
    "@babel/plugin-proposal-optional-chaining": "^7.11.0",
    "@babel/polyfill": "^7.12.1",
    "@svgr/parcel-plugin-svgr": "^5.4.0",
    "@types/react": "^16.14.28",
    "antd": "^4.2.0",
    "axios": "^0.19.2",
    "codecept3-upgrade": "^1.0.5",
    "core-js-compat": "^3.9.1",
    "crypto-js": "^4.0.0",
    "dayjs": "^1.11.3",
    "deep-diff": "^1.0.2",
    "dexie": "^3.2.0",
    "dotenv": "^8.2.0",
    "expr-eval": "^2.0.2",
    "faker": "^5.5.3",
    "glob": "^7.1.6",
    "lodash.clonedeep": "^4.5.0",
    "lodash.get": "^4.4.2",
    "lodash.groupby": "^4.6.0",
    "lodash.isequal": "^4.5.0",
    "lodash.mapvalues": "^4.6.0",
    "lodash.maxby": "^4.6.0",
    "lodash.minby": "^4.6.0",
    "lodash.pick": "^4.4.0",
    "lodash.remove": "^4.7.0",
    "lodash.uniqwith": "^4.5.0",
    "mark.js": "^8.11.1",
    "moment": "^2.24.0",
    "node-schedule": "^2.0.0",
    "parcel": "^2.0.0",
    "pubsub-js": "^1.9.4",
    "query-string": "^6.13.1",
    "randomcolor": "^0.6.2",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-is": "^17.0.1",
    "react-router-dom": "^5.1.2",
    "react-use": "^17.3.1",
    "reactn": "^2.2.6",
    "socket.io-client": "^4.1.2",
    "stream-browserify": "^3.0.0",
    "string-template": "^1.0.0",
    "styled-components": "^5.0.1",
    "swr": "^1.2.1",
    "throttle-debounce": "^3.0.1",
    "ts-node": "^10.8.1",
    "typescript": "^4.7.3",
    "uuid": "^7.0.3"
  },
  "devDependencies": {
    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
    "@parcel/packager-raw-url": "^2.6.2",
    "@parcel/transformer-less": "^2.6.2",
    "@parcel/transformer-webmanifest": "^2.6.2",
    "@types/node": "^17.0.42",
    "@types/pubsub-js": "^1.8.3",
    "babel-eslint": "^10.1.0",
    "buffer": "^6.0.3",
    "codeceptjs": "^3.0.4",
    "crypto-browserify": "^3.12.0",
    "eslint": "^7.5.0",
    "eslint-config-prettier": "^6.11.0",
    "eslint-plugin-babel": "^5.3.1",
    "eslint-plugin-codeceptjs": "^1.3.0",
    "eslint-plugin-import": "^2.22.0",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-react": "^7.20.3",
    "eslint-plugin-react-hooks": "^4.1.2",
    "events": "^3.3.0",
    "husky": "^4.3.0",
    "less": "^3.11.1",
    "lint-staged": "^10.3.0",
    "parcel-plugin-static-files-copy": "^2.3.1",
    "prettier": "^2.1.1",
    "puppeteer": "^5.5.0"
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "*.js": "eslint --fix",
    "*.{js,css,md}": "prettier --write"
  },
  "resolutions": {
    "@babel/preset-env": "7.13.8"
  }
}
`
Could help me someone please? I have not any much experience with parcel 
I will be grateful with any help
Thank you very much for your time.
BR!

**Comment by mischnic** [comment]
Strange, that should have been fixed by https://github.com/parcel-bundler/parcel/pull/8235

**Reply by frantwo** [reply]
Hello!
Can I do something to resolve it or it is necessary to attach more info to detail the problem?
I will do whatever that you recommend me :) 
Thanks a lot for your quick feedback and for your time!! this is amazing community!
BR
